packages/grafana-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { NavModelItem } from '@grafana/data';
+import { useTheme } from '../../themes';
+import { getTabsStyle } from './styles';
+import { cx } from 'emotion';
+
+export interface Props {
+  main: NavModelItem;
+  customCss?: string;
+  onChangeTab: (id: number) => void;
+}
+
+export const Tabs: FC<Props> = ({ main, onChangeTab, customCss }) => {
+  const theme = useTheme();
+  const tabsStyles = getTabsStyle(theme);
+
+  return (
+    <ul className={`${tabsStyles.tabs} ${customCss && customCss}`}>
```

Comment:
We usually name this property className?: string; and use cx to compose class names (ie customCss => className)

Reply:
Ah, right. Copy/paste error